packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -24,8 +25,12 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
     const { openOrganizationProfile, openCreateOrganization } = useCoreClerk();
     const { organization: currentOrg } = useCoreOrganization();
     const { isLoaded, setActive } = useCoreOrganizationList();
+    const router = useRouter();
     const {
       hidePersonal,
+      __unstable_manageBillingUrl,
```

Comment:
@nikosdouvlis What's the latest approach for `__unstable` props? Do we need to update `clerk.ts` in the `@clerk/types` package as well?

Reply:
We shouldn't type these anywhere as we don't want these props to visible at all - they're just going to get propagated as simple runtime values